auditwheel/patcher.py

```diff
@@ -61,6 +61,10 @@ def set_rpath(self,
                   file_name: str,
                   rpath: str) -> None:
 
+        old_rpath = check_output(['patchelf', '--print-rpath',
```

Comment:
I think this logic should not live in this method as other implementations of `ElfPatcher` in the future will not work consistently (my understanding is that having to append the `RPATH` is not a `patchelf` quirk, we should always do it). `set_rpath` should just set the provided `RPATH`.

We should either have `get_rpath` and `append_rpath` methods on the interface, or have an `append` flag which can trigger this behaviour.

Reply:
Sounds good -- if you don't have a particular preference I'll extend the interface.

Reply:
How's this?